src/docfx/build/xref/XrefMap.cs

```diff
@@ -2,28 +2,29 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
-using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.IO;
 using System.Linq;
-using System.Text;
-using System.Text.RegularExpressions;
 using System.Threading;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Docs.Build
 {
     internal class XrefMap
     {
         // TODO: key could be uid+moniker+locale
-        private readonly IReadOnlyDictionary<string, List<(InternalXrefSpec specs, Document file)>> _internalXrefMap;
-        private readonly IReadOnlyDictionary<string, Lazy<XrefSpec>> _externalXrefMap;
+        private readonly IReadOnlyDictionary<string, List<Lazy<IXrefSpec>>> _map;
```

Comment:
add a test case for same uid exist in both external xref map and internal xref map